### ISSUE TITLE
Proptypes issues with new RN 48

### ISSUE THREAD

**Issue by mmaschenko** [Andr3wHur5t/react-native-keyboard-spacer#61]
After i upgraded react native to last version, react-native-keyboard-spacer started to crash application because after react changes Proptypes is now can not be imported from react directly. You need install another package as described here https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html. I was able to fix it by changing package files directly, but it is bad idea for a lot of reasons

**Comment by hmziqrs** [comment]
fixed this issue by editing KeyboardSpacer.js

`import React, { Component } from 'react';
import PropTypes from 'prop-types';
`

**Comment by WillyRamirez** [comment]
@hackerhgl  Thnx, worked.  
PR tho?

**Comment by hmziqrs** [comment]
@WillyRamirez They are too lazy to do accept a simple PR

https://github.com/Andr3wHur5t/react-native-keyboard-spacer/pull/58

**Comment by WillyRamirez** [comment]
@hackerhgl  yeah i saw, OP is no longer supporting this package. Luckily it's just one file, i took it out and added it to my own components folder. Alternatively we could fork it, merge all PR

**Comment by NachtRitter** [comment]
@Andr3wHur5t please merge PRs and make new release. If you don't want to maintance this repo, just give rights to someone else.

**Comment by Andr3wHur5t** [comment]
I apologize if my response time is slow.

Every PR is tested with multiple versions of react native to see what minimal supported version is to consider all the apps which use this that haven’t upgraded; breaking existing apps rather than upgraded ones (which haven’t released) would be bad also.

I would love to have automated tests to streamline this process but it has proven difficult with the feature support (such as hardware keyboards). As such I do manual testing which takes time.

As for adding another maintainer I’m open to this assuming they consider down stream consequences (requests welcome); but as of yet no one has offered. 

I’ve recently pushed a out which works on the most recent version of RN. If you are still seeing this issue please post a repro so I can verify.
